src/sensors/src/ThreeAxisAngularAccelerometerSensor.cpp

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (C) 2017 Fondazione Istituto Italiano di Tecnologia
+ * Author: Silvio Traversaro
+ * email:  silvio.traversaro@iit.it
+ * Permission is granted to copy, distribute, and/or modify this program
+ * under the terms of the GNU General Public License, version 2 or any
+ * later version published by the Free Software Foundation.
+ *
+ * A copy of the license can be found at
+ * http://www.robotcub.org/icub/license/gpl.txt
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details
+ */
+
+
+# include "iDynTree/Core/LinearMotionVector3.h"
```

Comment:
This and a lot of other problems of this class are actually shared with all other sensor classes. I will open a new issue to cleanup all of them, instead of leaving them in a inconsistent state.